### PR TITLE
Setting github token when pushing on  master

### PR DESCRIPTION
**Pull request by knzeng-e**
### Summary

|             |   |
|-------------|---|
| Description |   |
| Jira issue  |   |

#### Type of request

- [ ] Bug fix (non-breaking change which fixes an issue)

### List of Features

- setting token to add write access to CI

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -63,6 +63,8 @@ jobs:
 
       - name: Deploy on master branch
         if: github.ref == 'refs/heads/master'
+        env:
+            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (jrhender) [inline .github/workflows/deploy.yml:67]: How is `GITHUB_TOKEN` then used by the script?

**Comment by jrhender** [comment]
Btw you can remove tags to rerun the build by running `git push --delete origin v0.6.0`

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (knzeng-e) [reply .github/workflows/deploy.yml:67]: The aim to give to ci action write access by enable write permission on GITHUB_TOKEN scope (in settings)

**Comment by knzeng-e** [comment]
> Btw you can remove tags to rerun the build by running `git push --delete origin v0.6.0`

thanks a lot for the tip !

**Review by jrhender — DISMISSED** [review]
Let's give it a try 👍

**Pull request closed without merging**